### ISSUE TITLE
`no-member` false positive for azure.servicebus

### ISSUE THREAD

**Issue by epomatti** [PyCQA/pylint#3121]
### Steps to reproduce
1. Create an environment with Python 3.7
2. Install Pylint
3. Import Azure Service Bus SDK for Python
4. Copy the example code from Microsoft to dequeue messages
5. An error will be displayed by Pylint, but the code runs normally.

To a complete setup run it by cloning this repository:

https://github.com/epomatti/python-api.git

### Current behavior

Pylint gives the following error:

```log
queue_receiver: Receiver
Instance of 'BaseHandler' has no 'fetch_next' member
pylint(no-member)
```

### Expected behavior

The error should not occur because the code is correct. It runs fine without Pylint, and with Python 3.6 the error is not displayed by Pylint.

![image](https://user-images.githubusercontent.com/6530769/65538964-dfc75780-dede-11e9-937f-99b0bdc0ce44.png)

### pylint --version output

```shell
$ pylint --version output
pylint 2.4.0
astroid 2.3.0
Python 3.7.4 (default, Sep  2 2019, 20:47:34) 
[GCC 7.4.0]
```

**Comment by thomkeh** [comment]
Probably related to #3116

**Comment by PCManticore** [comment]
Thanks, I can reproduce the issue as well.

**Comment by epomatti** [comment]
> 
> 
> Probably related to #3116

Well, maybe. In my case I have problems with Python 3.7 only, it works on 3.6. In the thread you linked he is having problem with 3.6.

**Comment by michaelf-lumedic** [comment]
Checking in 7 months later, Have the same issue, on the same line of code.
It did work until I put the method in a class.

**Comment by Pierre-Sassoulas** [comment]
Closing because it's lib specific and not a very well known one, stale, and hard to reproduce.
